android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -27,6 +28,9 @@ class AppLovinMAXAdView
         extends ReactViewGroup
         implements MaxAdListener, MaxAdViewAdListener, MaxAdRevenueListener
 {
+    /// Static Instance for Amazon
```

Comment:
No need for this comment since it's obvious from the line of code what it is (we also never do `///` for comments in our codebase)